### DISCUSSION TITLE
Brush size slider is highly unresponsive

### DISCUSSION THREAD

**Discussion by tryashtar** [Mojang/minecraft-editor#35] [Bugs]
When you click the brush size slider, it takes upwards of one second for the value to actually update. Dragging it around is similarly delayed.

https://user-images.githubusercontent.com/26075577/223856123-c9804706-7ac6-481f-b55d-7cd91a1a4cbe.mp4

**Comment by minecraftchloe** [comment]
It looks like your FPS is super low - what other processes do you have running and/or are other actions in Minecraft similarly slow? Made a note for a bigger investigation around UI performance.
